apps/condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotifications.js

```diff
@@ -0,0 +1,288 @@
+const dayjs = require('dayjs')
+const { get, uniq } = require('lodash')
+
+const conf = require('@core/config')
+const { getSchemaCtx } = require('@core/keystone/schema')
+
+const { loadListByChunks } = require('@condo/domains/common/utils/serverSchema')
+const { COUNTRIES, DEFAULT_LOCALE } = require('@condo/domains/common/constants/countries')
+
+const { Meter, MeterReading, MeterResource } = require('@condo/domains/meter/utils/serverSchema')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+const {
+    METER_SUBMIT_READINGS_REMINDER_TYPE,
+    METER_VERIFICATION_DATE_EXPIRED_TYPE,
+} = require('@condo/domains/notification/constants/constants')
+
+
+const { rightJoin, joinResidentsToMeters } = require('@condo/domains/meter/tasks/sendVerificationDateReminder')
+
+const log = (message, intentChar = null, intentRepeatTimes = 10) => {
```

Comment:
We are using pino json logs - because they are stored in elasatic db

Reply:
ok